cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -8,16 +8,64 @@ toc::[]
 Creating deployment configurations, build configurations, service routes and other {product-title} elements are all automated by {odo-title}.
 
-Existing tools such as `oc` are more operations-focused and require a deep understanding of Kubernetes and OpenShift concepts. {odo-title} abstracts away complex Kubernetes and OpenShift concepts allowing developers to focus on what is most important to them: code.
-
+Existing tools such as `oc` are more operations-focused and require a deep understanding of Kubernetes and {product-title} concepts. {odo-title} abstracts away complex Kubernetes and {product-title} concepts allowing developers to focus on what is most important to them: code.
 
+[id="odo-key-features"]
 == Key features
 
 {odo-title} is designed to be simple and concise with the following key features:
 
 * Simple syntax and design centered around concepts familiar to developers, such as projects, applications, and components.
 * Completely client based. No server is required within the {product-title} cluster for deployment.
-* Official full support of Node.js and Java components.
-* Partial compatibility with many languages and frameworks such as Ruby, Perl, PHP, and Python. Run odo catalog list components to see the list of supported and unsupported components.
+* Official support for Node.js and Java components.
+* Partial compatibility with languages and frameworks such as Ruby, Perl, PHP, and Python. 
 * Detects changes to local code and deploys it to the cluster automatically, giving instant feedback to validate changes in real time.
-* Lists all the available components and services from the {product-title} cluster.
\ No newline at end of file
+* Lists all the available components and services from the {product-title} cluster.
+
+[id="odo-supported-languages-and-images"]
+=== Officially supported languages and corresponding container images
+
+.Supported container images
+[options="header"]
+|====
+|Node.js|Java
+|`centos/nodejs-8-centos7`|`redhat-openjdk-18/openjdk18-openshift`
+|`rhoar-nodejs/nodejs-8`|`openjdk/openjdk-11-rhel8`
+|`rhoar-nodejs/nodejs-10`|`openjdk/openjdk-11-rhel7`
+|`bucharestgold/centos7-s2i-nodejs`|
+|`rhscl/nodejs-8-rhel7`|
+|`rhscl/nodejs-10-rhel7`|
+|====
+
+[id="odo-listing-available-images"]
+==== Listing available container images
+
+[NOTE]
+====
+The list of available container images is sourced from the cluster's internal container registry and external registries associated with the cluster.
+====
+
+To list the available components and associated container images for your cluster:
+
+. Log in to the {product-title} cluster with {odo-title}:
++
+----
+$ odo login -u developer -p developer
+----
+
+. List the available {odo-title} supported and unsupported components and corresponding container images with `odo catalog list components`:
```

Comment:
Not critical, but I don't think the "with `odo catalog list components`" part is necessary. You show the command in the code block below.